src/components/form-pages/Catch.js

```diff
@@ -74,25 +115,39 @@ export default function Catch({ handleAddCatch, catches }) {
               </Part>
               <Notes>
                 <label htmlFor="notes">Notes</label>
-                <Input id="notes" name="notes" type="text" maxLength={300} />
+                <Input
+                  onChange={handleChange}
+                  value={values.notes}
+                  id="notes"
+                  name="notes"
+                  type="text"
+                  maxLength={300}
+                />
               </Notes>
             </Fieldset>
             <Button
               text={'Add Catch'}
               isDark={true}
-              onClick={handleAddCatch}
+              onClick={() => {
+                handleAddCatch(values);
+                setValues(initialValues);
+              }}
               icon={<HiPlusCircle />}
             />
             <CatchList>
               <span>Your catches:</span>
-              {catches.map((data, index) => (
-                <Catches key={index}>
-                  <span>{index + 1}.</span>
-                  <span>{data.species}</span>
-                  <span>{data.length} cm</span>
-                  <HiOutlineTrash size={25} color={'#a2c36c'} />
-                </Catches>
-              ))}
+              {catches ? (
+                catches.map((data, index) => (
+                  <Catches key={index}>
+                    <span>{index + 1}.</span>
+                    <span>{data.species}</span>
+                    <span>{data.length} cm</span>
+                    <HiOutlineTrash size={25} color={'#a2c36c'} />
```

Comment:
Maybe remove this delete icon for now? You can add it later if you decide to add a delete functionality.